app/src/main/java/is/xyz/mpv/MPVView.kt

```diff
@@ -220,6 +220,10 @@ internal class MPVView(context: Context, attrs: AttributeSet) : SurfaceView(cont
     val hwdecActive: Boolean?
         get() = MPVLib.getPropertyString("hwdec-current") != "no"
 
+    var playbackSpeed: String?
+        get() = MPVLib.getPropertyString("speed").toFloat().toString()
```

Comment:
Why do you convert a string to float then back to string again? Also, shouldn't the property be a float?

Reply:
There were no property methods for double properties, and the reconversion was for getting rid of excessive zeroes. I added those methods now.